maa-cli/src/config/mod.rs

```diff
@@ -212,6 +215,81 @@ pub fn convert(file: &Path, out: Option<&Path>, ft: Option<Filetype>) -> Result<
     }
 }
 
+pub fn cleanup(inquire: bool) -> Result<(), io::Error> {
+    let state = dirs::state().to_path_buf();
+    let debug = dirs::log().to_path_buf();
+
+    let tarsh_list = vec![
+        (
+            "maa-cli cache (e.g. \x1B[33mupdate and copliot files\x1B[0m)",
+            vec![dirs::cache().to_path_buf()],
+        ),
+        (
+            "maa-core avatars cache",
+            vec![state.join("cache").join("avatars")],
+        ),
+        (
+            "maa-core log files (\x1B[33masst.log and asst.bak.log\x1B[0m)",
+            vec![debug.join("asst.log"), debug.join("asst.bak.log")],
+        ),
+        (
+            "maa-core screenshot cache",
+            vec![
+                debug.join("drops"),
+                debug.join("map"),
+                debug.join("other"),
+                debug.join("Roguelike"),
+            ],
+        ),
+    ];
```

Comment:
I think we can also clear log files created by `maa-cli` and stored in default log path.

Reply:
I didn't find default log file of maa-cli. Where can I find it?

Reply:
https://github.com/MaaAssistantArknights/maa-cli/blob/311b24e0bd39482785164d18946310493063d147/maa-cli/src/main.rs#L307-L314

Reply:
Sorry for confusing, this location is used only when run with `--log-file`, and will not generate by default.

Reply:
thanks